types.go

```diff
@@ -607,7 +607,7 @@ type SharedRunnersSettingValue string
 // https://docs.gitlab.com/ee/api/groups.html#options-for-shared_runners_setting
 const (
 	EnabledSharedRunnersSettingValue                  SharedRunnersSettingValue = "enabled"
-	DisabledWithOverrideSharedRunnersSettingValue     SharedRunnersSettingValue = "disabled_with_override"
```

Comment:
I wonder if it makes sense to leave the deprecated setting in here just to be more graceful and be backwards compatible. You could mark the setting as deprecated:

```golang
// Deprecated: DisabledWithOverrideSharedRunnersSettingValue is deprecated in favor of DisabledAndOverridableSharedRunnersSettingValue  
DisabledWithOverrideSharedRunnersSettingValue       SharedRunnersSettingValue = "disabled_with_override"
DisabledAndOverridableSharedRunnersSettingValue   SharedRunnersSettingValue = "disabled_and_overridable"
```

Reply:
That's a good point! I added it as deprecated 👍